app/javascript/packs/application.js

```diff
@@ -18,11 +18,11 @@ import 'bootstrap';
 // Fontawesome: https://fontawesome.com/
 import "@fortawesome/fontawesome-free/js/all";
 
+import * as polylabel from 'polylabel';
+global.polylabel = polylabel
```

Comment:
I'm not quite sure about this import.

1. Wouldn't it be better to only import it where we need it (`leafletMap.js`)? With this, we wouldn't need to pollute the global scope.
2. I think the `global` variable is a convention from NodeJs. In a browser, you should use `window` like `window.polylabel = polylabel`. [References](https://developer.mozilla.org/en-US/docs/Glossary/Global_object)

Reply:
I will go with your first suggestion